docs/en/operation/StressBench.md

```diff
@@ -111,3 +111,69 @@ This process continues until the max throughput is discovered.
 This algorithm uses many iterations and time to determine the throughput, but it produces a more consistent result, and the actual number is meaningful.
 If the resulting throughput is X, that means if the clients produce X throughput, the server can achieve that.
 It also means if the clients produce more than X throughput, like X+N, the server will not be able to achieve X+N throughput.
+
+
+## Fuse IO Stress Bench
+The Fuse IO Stress Bench is a tool to measure the reading performance of Alluxio Fuse-based POSIX API.
+
+### Procedure
+
+#### 0. Prerequisite
+A running Alluxio cluster is required. The Alluxio path for testing needs to be mounted to the local file system. To prevent Fuse cache affecting
+test accuracy, it is suggested to use different mount points for writing test files and read. See 
+[FUSE-based POSIX API](https://docs.alluxio.io/os/user/stable/en/api/POSIX-API.html) for details on how to mount Alluxio to local file system.
+
+#### 1. Write test files
+Write the test files by running the benchmark with `--operation Write` into the mount point for writing files. More specifically, run 
+
+`$ bin/alluxio runClass alluxio.stress.cli.fuse.FuseIOBench --operation Write --local-path /path/to/writing/mount/point ...`
+
+#### 2. List out test files (optional)
+List out the test files by running the benchmark with `--operation ListFile` at the mount point for reading files. More specifically, run
+
+`$ bin/alluxio runClass alluxio.stress.cli.fuse.FuseIOBench --operation ListFile --local-path /path/to/reading/mount/point`
+
+This operation will cache the file metadata, so that the metadata operation while reading won't affect reading throughput accuracy. To enable metadata cache,
+`alluxio.user.metadata.cache.enabled` needs to be set to `true` in `${ALLUXIO_HOME}/conf/alluxio-site.properties` before launching Alluxio cluster. 
+
+#### 3. Read test files 
+Read the test files by running the benchmark with `--operation {LocalRead, RemoteRead, ClusterRead}` from the reading mount point. For example, run
+
+`$ bin/alluxio runClass alluxio.stress.cli.fuse.FuseIOBench --operation LocalRead --local-path /path/to/reading/mount/point ...`
+
+### Single-node mode test
```

Comment:
single node mode is what we talking about to distinguish from cluster mode
but from user perspective, better to rephrase into 
### Testing targets
#### Single node testing
#### Cluster testing